analyzer/config/docker/testdata/deny.rego

```diff
@@ -0,0 +1,13 @@
+package testdata 
+
+denylist = [
+  "foo"
+]
+
+deny[msg] {
+  input[i].Cmd == "from"
+  val := input[i].Value
+  contains(val[i], denylist[_])
+
+  msg = sprintf("deny: image found %s", [val])
+}
```

Comment:
We need to return structured messages as follows. But even if a rule returns only "msg", we have to handle it. So, both cases are necessary. We can keep `violation.rego` as is and change this rego to return a structured message. Also, we might have the case missing some fields like `{"msg": "foo", "id": "RULE-100"}`. In this case, `type` and `severity` are missing.

```suggestion
deny[res] {
  input[i].Cmd == "from"
  val := input[i].Value
  contains(val[i], denylist[_])

  res = {"type": "Docker Security Check", "msg": sprintf("deny: image found %s", [val]), "severity": "HIGH", "id": "RULE-100"}
}
```

Reply:
alright will try to add more cases like this.